scalableminds/webknossos
# Fix initial min/max values of a histogram of a color layer

It is possible to configure only the `min` and `max` for a color layer in the view configurations of a dataset. This causes that the backend to send the default values for the `intensityRange` for the color layer. 
IIn case the `min` is higher than the sent intensity minimum or the max is lower than the sent intensity maximum the frontend got into an inconsistent state where the `intensityRange` is out of bounds considering the `min` and `max`.

A short solution to this is to force the intensityRange to be within the range given by `min` and `max`.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
 - Go to the settings of a dataset and then to the "View Configurations"-tab.
 - There choose a (e.g. uint8) color layer and set `min` to 100 and `max` to 150.
 - Then make the dataset publically available under the "Sharing & Permissions"-tab, copy the sharing link and save the settings.
 - Then open the dataset with the public/sharing link in a private browser tab.
 - The rendered color layer should render data like the histogram slider is at `[100,150]`.   

### Issues:
- fixes #5694

------
